daemon/algod/api/algod.oas2.json

```diff
@@ -3964,11 +3977,42 @@
         "exec-trace": {
           "$ref": "#/definitions/SimulationTransactionExecTrace"
         },
+        "txn-initial-state": {
+          "$ref": "#/definitions/SimulateTxnInitialState"
+        },
         "unnamed-resources-accessed": {
           "$ref": "#/definitions/SimulateUnnamedResourcesAccessed"
         }
       }
     },
+    "SimulateTxnInitialState": {
+      "description": "A transaction's initial state during simulation.",
+      "type": "object",
+      "properties": {
```

Comment:
I don't understand the motivation of this structure. It's sending back the things that were in the transaction, mostly.

Sending the approval hash has some value, since it's not in the txn itself. But I would expect that to appear in some sort of `AppState` object, along with globals.